Configuration/PyReleaseValidation/scripts/runTheMatrix.py

```diff
@@ -56,7 +56,9 @@ def runSelected(opt):
                      136.731, #2016B Photon data
                      140.53, #2011 HI data
                      1330, #Run2 MC Zmm
-                     135.4 #Run 2 Zee ttbar
+                     135.4, #Run 2 Zee ttbar
+                     11024.0, #2023LReco ttbar (w/ HGCal)
+                     11224.0  #2023GReco ttbar (flat tracker, Run2 calo)
```

Comment:
@slava77 @davidlange6 we should make a decision here - is it okay to put Phase2 (and additionally 2017?) in the limited matrix, or should I update just the bot's list of extra PR tests?